sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/New-SPOListFileVersionBatchDeleteJob.md

```diff
@@ -101,6 +147,49 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExcludeDefaultPolicy
+Indicates whether to trim versions based on the default version history limits.
+
+> [!NOTE]
+> This feature is currently in preview and may not be available in your tenant.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: ByPolicy
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FileTypes
+An array of file type names to trim versions. The supported file type names are:
+
+- Audio
+
+- OutlookPST
+
+- Video
+
+
+
+
```

Comment:
remove extra blank lines. you also don't need a blank line between the items themselves